system-tests/test/component_testing_spec.ts

```diff
@@ -0,0 +1,62 @@
+import systemTests from '../lib/system-tests'
```

Comment:
These are the system tests that verify the `cypress.config.js` we generate is valid and works correctly.